tests/errors.js

```diff
@@ -0,0 +1,139 @@
+import test from 'ava';
+import events from 'events';
+import httpMocks from 'node-mocks-http';
+import _ from 'lodash';
+
+import routes from '../lib/errors/routes';
+
+const EventEmitter = events.EventEmitter;
+
+// used in a promise below
+let nextCalled = false; // eslint-disable-line no-unused-vars
+
+const get = (req) => {
+  if (req === 'env') {
+    return 'development';
+  }
+
+  return false;
+};
+
+// application fake object
+const app = {
+  get,
+};
+
+/**
+ * Next
+ * @param  {object} err - the error object
+ * @returns {object} nextCalled - the error object
+ */
+const next = (err) => {
+  nextCalled = err;
+
+  return nextCalled;
+};
+
+const err = {
+  message: '',
+  safe: '/content',
+  status: 404,
+};
+
+//////////////////////////////
+// Routes - 404
+//////////////////////////////
+test.cb('404 error route', t => {
+  const request = httpMocks.createRequest({
+    method: 'GET',
+    url: '/some/madeup/icky/path',
+  });
+
+  request._setSessionVariable('404', {});
+
+  _.set(request.session, '404', {
+    message: 'Something bad happened',
+    safe: '/refreshing/safe/path',
+  });
+
+  const response = httpMocks.createResponse({ eventEmitter: EventEmitter });
```

Comment:
Can we keep our `const`s together if they don't have internal dependencies on each other?